apps/backend/src/routes/home/+page.svelte

```diff
@@ -0,0 +1,133 @@
+<!-- JavaScript code -->
+<script>
+    // Your Svelte script here
+</script>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <script src="https://kit.fontawesome.com/c850f0d795.js" crossorigin="anonymous"></script>
+</head>
```

Comment:
Not allowed in svelte

Reply:
Removed <head> tag and its contents. Fixed the font awesome by using svg